engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
+title: "Logentries logging driver"
+description: "Describes how to use the logentries logging driver."
+keywords: ["logentries, docker, logging, driver"]
+---
+
+# Logentries logging driver
```

Comment:
There was drift between the title in the metadata and the displayed title that meant there wasn't a source of truth for titles anymore. Additionally there was inconsistency in how titles were written and cased. Finally, the ability to converge on metadata being the source of truth for titles meant that a dependable title was detachable from the doc for injection into other areas like HTML layouts, auto-complete boxes, etc.